api-spec/transactions-api.yaml

```diff
@@ -611,6 +611,7 @@ components:
         - CLOSURE_ERROR
         - NOTIFIED_OK
         - NOTIFIED_KO
+        - NOTIFICATION_ERROR
```

Comment:
Add this status enumeration also into infra service openapi specifications

Reply:
Done in pagopa/pagopa-infra#834